### PR TITLE
Create notices at end of start.sh and add to the README

### PR DESCRIPTION
**Pull request by bart88**
This continues to address #8 but its not quite there yet. I need help fleshing out the README and it would be good to get someone to review.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,9 +5,11 @@ environment.
...
 
-It is assumed that you have docker installed. For OS X and Windows users you will need to install the [docker toolbox](https://www.docker.com/products/docker-toolbox).
+It is assumed that you have docker installed. For __OS X__ and __Windows__ users you will need to install the [docker toolbox](https://www.docker.com/products/docker-toolbox).
 This will give you required tools to run docker through a vm layer ( docker-machine ).
 
+We follow the docker philosophy, meaning one container per application process. Because of this we use `docker-compose` to define all our containers.
+
 # Initial setup
 
 To use for your own projects, check out this repository and the remove the .git folder with:
```
Comment (pingers) [inline README.md]: S/repository and the/repository and then

File: README.md [hunk 2]
```diff
@@ -22,9 +24,36 @@ The next step is to start the docker environment with:
...
+```
+
+__Mac OS X users__:
+
+If you see this :
+```bash
+$ Cannot connect to the Docker daemon. Is the docker daemon running on this host?
+```
+
+You need to `eval` the docker-machine environment running. To find the name of the docker-machine(s) you are running :
```
Comment (pingers) [inline README.md:46]: Does this say anywhere to add yourself to docker group?

Comment (pingers) [inline README.md:46]: On my phone. Can't see much ☺

Comment (bart88) [inline README.md:46]: I don't think it does. Is this just assumed as part of a Unix docker setup ?

File: start.sh [hunk 3]
```diff
@@ -3,3 +3,11 @@
 . common.sh
 
 docker-compose up -d
+
+# Complete notice.
+notice "\nInstalling is complete!"
+notice "To view the containers created run:"
+notice "$ docker ps"
+notice "To create a utility container to build Drupal 8 run:"
```
Comment (pingers) [inline start.sh]: Maybe say to create a shell into a utility container

Comment (bart88) [inline start.sh]: 👍

**Comment by singularo** [comment]
I think this can probably be abandoned/deleted? Its well out of date now.
@bart88 ?

**Pull request closed without merging**